packages/redux-dynostore-react-redux-subspace/test/mapExtraState.test.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
These tests should be in the `subspaced.test.js` file.  You can put them in a nested `describe('mapExtraState tests', ...)` if you want to seperate them from the other tests

Reply:
Ok